src/main/java/util/CIDevelopLoaderImpl.java

```diff
@@ -14,6 +14,26 @@ public String getBasePathUrl() {
         }
     }
 
+    @Override
+    public String getApiKeys() {
+        String variable = System.getProperty("DEVELOP_BASE_PATH");
+        if (variable != null) {
+            return variable;
```

Comment:
For future reference and up to you but it is in general a better practice to have guard clause so we exit early if our code base fails so we don't unnecessarily traverse code. For example
```java
(if variable == null) { throw new AutomationException("Setup of variable DEVELOP_BASE_PATH failed, please pass in command or configure variable"); }
/** i can then execute the rest of my code **/
```

Also the  name of the variable is not very meaningful maybe `basePath` would have been more appropriate

Reply:
this class no longer exist